src/bindings/python/src/pyopenvino/graph/node_output.hpp

```diff
@@ -41,6 +41,26 @@ void regclass_graph_Output(py::module m, std::string typestring)
                 get_index : int
                     Index value as integer.
                )");
+    output.def("get_any_name",
+               &ov::Output<VT>::get_any_name,
+               R"(
+                Any tensor names associated with this output.
```

Comment:
Actually I will leave a note here that `get_any_name` return first (in lexicographical order) name associated with the output. In short - this is not "random" function. 
```suggestion
                One of the tensor names associated with this output.
                Note: first name in lexicographical order.
```